srv/isc-agent/plugins/tcp/http/main.py

```diff
@@ -13,6 +18,15 @@ def render_GET(self, request: Request):
         content = f"I am request #{self.numberRequests}\n"
         return content.encode("ascii")
 
+    def render_HEAD(self, request: Request):
+        request.setResponseCode(HTTPStatus.OK)
+        request.setHeader(b"Server", f"{PRODSTRING}")
+        request.setHeader(b"Access-Control-Allow-Origin", "*")
```

Comment:
You don't need to convert this to bytes. Also let's try to stick to single quotes for strings as much as possible